config/routes.rb

```diff
@@ -1,5 +1,12 @@
 Rails.application.routes.draw do
   devise_for :users
+  as :user do
+    get 'user/:id', :to => 'user#show', :as => :user_profile
```

Comment:
Com a chegada do Ruby 2 ficou muito incomum usar hash rocket, o que acham de usar a nova convenção

```suggestion
    get 'user/:id', to: 'user#show', as: :user_profile
```